src/WebJobs.Script/Diagnostics/Extensions/HostLoggerExtension.cs

```diff
@@ -56,7 +56,7 @@ public static class HostLoggerExtension
         private static readonly Action<ILogger, Exception> _debugerManagerUnableToUpdateSentinelFile =
             LoggerMessage.Define(
             LogLevel.Error,
-            new EventId(207, nameof(DebugerManagerUnableToUpdateSentinelFile)),
+            new EventId(207, nameof(DebuggerManagerUnableToUpdateSentinelFile)),
```

Comment:
Needs to be updated in the method name below, as well as at the call site calling that method. So it'll be 3 locations.

Reply:
Ah yes, have now updated in all locations.

Reply:
As I mentioned the call site of the method you renamed below also needs to be updated https://github.com/Azure/azure-functions-host/blob/711e83b65d0d80bbec7f2c6e640fe7daa8cb91b6/src/WebJobs.Script/Host/DebugManager.cs#L57